WcaOnRails/app/models/team.rb

```diff
@@ -4,6 +4,8 @@ class Team < ApplicationRecord
   has_many :team_members, dependent: :destroy
   has_many :current_members, -> { current }, class_name: "TeamMember"
 
+  default_scope -> { where(hidden: false) }
```

Comment:
As far as I know, it's generally not recommended to use `default_scope`, but I felt like it's the right solution as we may **really** not want to accidentally reveal a hidden team (e.g. once a new feature is added and someone doesn't remember to filter hidden teams out).